docs/portal_config.md

```diff
@@ -633,6 +633,8 @@ Below is an example, with inline comments describing what each JSON block config
     "studyRegistrationValidationField": "is_registered", // optional, the other of the extra field that is being added to metadata when a study is registered, to check if a study has been registered, because after loading data from MDS/AggMDS into Discovery page, the metadata category information is lost. Defaults to "is_registered"
     "studyRegistrationAccessCheckField": "registration_authz", // optional, the field that contains the value for Study Registration Request Access feature. Defaults to "registration_authz"
     "studyRegistrationUIDField": "appl_id", // optional, the field which can be used to uniquely determine a metadata record for Study Registration. Defaults to "appl_id"
+    "studyRegistrationFormDisclaimerField": null, //optional, the diclaimer text that appears under the submit button
+    on the study registration form. Defaults to null
```

Comment:
if you changes the defaults to `undefined`, also changes this

Reply:
For the markdown, would be replace `null` with an empty quote `""`? And remove the `Defaults to null`?

Reply:
you can put some dummy text in there like `"This is a disclaimer"` and this defaults to `undefined`